docs/src/Fulma/Modifiers/Colors.fs

```diff
@@ -65,7 +65,7 @@ let view =
     Render.docPage [
         Render.contentFromMarkdown
             """
-# Modifiers
+# Modifiers - Colors & Shades
 
 ## Colors
 
```

Comment:
color & shade isn't technically a modifier you can add.
you add the modifiers BackgroundColor and TextColor, which can be a color or a shade.
and you can add them to components also if I am not mistaken.

Reply:
Yes but I don't have a good small name :)

I kind of mimic Bulma docs here. They speak about `Color helpers`